readme.md

```diff
@@ -26,7 +26,7 @@ When used with a kit, everything is available, according to that kit's configura
 - Font Awesome Pro
 - Font Awesome 6
 
-    Including the Sharp Solid style, when using a Pro kit with Font Awesome 6.2.0 or newer!
+    Including the Sharp Regular & Light styles, when using a Pro kit with Font Awesome 6.4.0 or newer!
```

Comment:
again FA 6.3.0 is the min version for Sharp Regular, while 6.4.0 is the min version for Sharp Light. This sentence doesn't distinguish them, so it sounds like they both first became available in 6.4.0.

Reply:
@mlwilkerson would it make sense just to remove that sentence?

Reply:
Actually, yeah. Removing it, it's not necessary now